src/hooks/useSuggestions.ts

```
@@ -0,0 +1,148 @@
+                    }),
+                }),
+            }),
+            update: async (data: any) => ({
+                eq: async () => ({ error: null }),
+            }),
+        }),
+    };
+};
+
+export const useSuggestions = (clubId: string) => {
+    const supabase = useSupabase();
```

ESLint

[error] Replace `····` with `··`

(prettier/prettier)